integration_testing/features/coach/coach-edit-quiz.feature

```diff
@@ -4,6 +4,7 @@ Feature: Coach edits quizzes
   Background:
     Given I am signed in to Kolibri as a coach user
       And there is a quiz <quiz> created previously
+      And the quiz is started
```

Comment:
I don't think this line is necessary, coach can make these changes even if the quiz in not started.

Reply:
This is because later on, I instruct the tester to verify that the learner can indeed take the quiz and the learner can only see the quiz if it is started / active.